[2023-05-05T09:45:17.531+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chatgpt_dag.delete_file scheduled__2023-05-05T09:40:00+00:00 [queued]>
[2023-05-05T09:45:17.536+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chatgpt_dag.delete_file scheduled__2023-05-05T09:40:00+00:00 [queued]>
[2023-05-05T09:45:17.536+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-05-05T09:45:17.536+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-05-05T09:45:17.536+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-05-05T09:45:17.546+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): delete_file> on 2023-05-05 09:40:00+00:00
[2023-05-05T09:45:17.549+0000] {standard_task_runner.py:55} INFO - Started process 392899 to run task
[2023-05-05T09:45:17.551+0000] {standard_task_runner.py:82} INFO - Running: ['airflow', 'tasks', 'run', 'chatgpt_dag', 'delete_file', 'scheduled__2023-05-05T09:40:00+00:00', '--job-id', '4670', '--raw', '--subdir', 'DAGS_FOLDER/chatgpt_dag.py', '--cfg-path', '/tmp/tmpmkeggdmh']
[2023-05-05T09:45:17.551+0000] {standard_task_runner.py:83} INFO - Job 4670: Subtask delete_file
[2023-05-05T09:45:17.582+0000] {task_command.py:389} INFO - Running <TaskInstance: chatgpt_dag.delete_file scheduled__2023-05-05T09:40:00+00:00 [running]> on host trivia-machine.us-central1-a.c.triviapractice-381517.internal
[2023-05-05T09:45:17.628+0000] {taskinstance.py:1516} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=richc
AIRFLOW_CTX_DAG_ID=chatgpt_dag
AIRFLOW_CTX_TASK_ID=delete_file
AIRFLOW_CTX_EXECUTION_DATE=2023-05-05T09:40:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-05-05T09:40:00+00:00
[2023-05-05T09:45:17.632+0000] {logging_mixin.py:137} INFO - /home/rich2/airflow/scripts/chatgpt_output/oscars_2023-05-05_09-45-13.json deleted successfully
[2023-05-05T09:45:17.632+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-05-05T09:45:17.635+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=chatgpt_dag, task_id=delete_file, execution_date=20230505T094000, start_date=20230505T094517, end_date=20230505T094517
[2023-05-05T09:45:17.682+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-05-05T09:45:17.689+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
